src/server/controllers/events.controller.js

```diff
@@ -4,6 +4,14 @@ const omit = require('lodash/omit');
 const eventsService = require('../services/events.service');
 const userService = require('../services/user.service');
 
+function isAuthenticated(request, response, next) {
```

Comment:
What do you mean?
- Use that file to store this function?
- Use `isAuthenticated` function from `src/server/utils/security-utils.js`?